lib/test.js

```diff
@@ -975,10 +975,11 @@ class Test extends Base {
 
   get fullname () {
     const main = process.argv[1] || ''
-    return (this.parent ? this.parent.fullname
+    const name = (this.parent ? this.parent.fullname
       : main.indexOf(cwd) === 0 ? main.substr(cwd.length + 1)
-      : path.basename(main)).replace(/\\/g, '/') +
-      ' ' + (this.name || '').trim()
+      : path.basename(main)) + ' ' + (this.name || '').trim()
+    
+    return name.replace(/\\/g, '/')
```

Comment:
Oh, yes you are right. Not sure why the current one doesnt work even though the replacement. Whats a more suitable place to handle this? Any hints are appreciated.

Reply:
Yeah, I'm guessing that maybe the name is getting set somewhere in the `Spawn` subclass perhaps?  I'd been using the Windows support on Travis briefly, but ran into too many issues around signal and subprocess differences, so I think the only way forward is for me (or someone) to spin up a Windows VM and get all the tests passing locally first.  The turnaround is just too slow to do with a remote CI system otherwise, and a known-failing test is worse than no test at all.